### ISSUE TITLE
Some questions about train snn_angular_velocity

### ISSUE THREAD

**Issue by WeijunDing1** [uzh-rpg/snn_angular_velocity#10]
hi I'm a student from Hongkong, I'm trying to change this code's dataset and loss function to train a new snn model, but I cannot backpropagate and update the parameters to minimize the loss function. Could you please tell me how can i do this well? Thank you very much.
